website/handlers/redirect.ts

```diff
@@ -4,13 +4,16 @@ import { isFreshCtx } from "./fresh.ts";
 export interface RedirectConfig {
   to: string;
   type?: "permanent" | "temporary";
+  keepQueryParameters?: boolean;
```

Comment:
```suggestion
  discardQueryParameters?: boolean;
```

Current behavior keeps query parameters if they are present.

We should keep current behavior if this new parameter is `undefined`.